src/Examine/LuceneEngine/Providers/LuceneIndex.cs

```diff
@@ -1240,12 +1381,44 @@ private IndexWriter CreateIndexWriter()
         protected virtual IndexWriter WriterFactory(Directory d)
         {
             if (d == null) throw new ArgumentNullException(nameof(d));
-            var writer = new IndexWriter(d, FieldAnalyzer, false, IndexWriter.MaxFieldLength.UNLIMITED);
 
-            // clear out current scheduler and set the error logging one
-            writer.MergeScheduler.Dispose();
-            writer.SetMergeScheduler(new ErrorLoggingConcurrentMergeScheduler(Name,
-                (s, e) => OnIndexingError(new IndexingErrorEventArgs(this, s, "-1", e))));
+            IndexWriter writer;
+            if (_directory is ExamineDirectory examineDirectory)
```

Comment:
Would like to explore using SnapshotDeletePolicy instead of having to do all of this extra work